src/OpenLoco/Windows/Vehicle.cpp

```diff
@@ -942,6 +944,15 @@ namespace OpenLoco::Ui::Vehicle
             return self;
         }
 
+        static void cloneVehicle(window* self)
+        {
+            static loco_global<uint16_t, 0x0113642A> _113642A;
+            auto head = Common::getVehicle(self);
+            GameCommands::do_80(head->head);
+            auto newHead = ThingManager::get<Vehicles::VehicleHead>(ThingManager::get<Vehicles::VehicleBase>(_113642A)->getHead());
+            OpenLoco::Ui::Vehicle::Details::open(newHead);
```

Comment:
I believe this should check for a failed state before retrieving `newHead` and opening its vehicle window?